src/components/Tool/FixMailingAddresses/FixMailingAddresses.test.tsx

```diff
@@ -108,13 +108,24 @@ describe('FixMailingAddresses', () => {
     ).not.toBeInTheDocument();
   });
 
-  it('should count total contacts', async () => {
+  it('should count total contacts and startDate', async () => {
     const { queryByTestId, getByText } = render(
       <Components
         mocks={{
           InvalidAddresses: {
             contacts: {
-              nodes: [{}, {}, {}],
+              nodes: [
+                {
+                  id: contactId,
+                  name: 'Baggins, Frodo',
+                  status: null,
```

Comment:
You can probably remove these mocked fields and just keep the `addresses` one here and in the next test.